ldapadmin/ldapadmin.properties

```diff
@@ -53,6 +53,13 @@ subject.change.password=[geOrchestra] Update your password
 subject.account.uid.renamed=[geOrchestra] New login for your account
 warnUserIfUidModified=true
 
+# Email proxy configuration
+proxyFromAddress=no-reply@georchestra.org
```

Comment:
more explicit keys would be great, eg `emailProxyFromAddress`, same for the other ones below.